### DISCUSSION TITLE
mini.files: bookmark persistence across nvim sessions

### DISCUSSION THREAD

**Discussion by proofer** [nvim-mini/mini.nvim#1251] [Ideas]
I would like a way to have mini.files bookmarks persist across nvim sessions. This would require on-disk storage, so I think it would have to be a configuration option.

**Comment by echasnovski** [comment]
The suggested way to do that is [present in examples](https://github.com/echasnovski/mini.nvim/blob/848c5e8f428faf843051768e0d56104cd02aea1f/doc/mini-files.txt#L522-L537). That is, create an autocommand for `MiniFilesExplorerOpen` event and set custom bookmarks inside it.
